### PR TITLE
Review your own project

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
This is my first review project in Microverse Program

# Recommendations: #

- Fix linter Errors if you have.
- Use HTML Semantic Tags
- Use the best Selector in CSS
- Don't use the same id for more than one tag

# Important for Reviewers who will review my review #
NOTE: This is the second time I am submitting my repository. You'll not find the first reviews in my pull request because after the first submission I make a big mess to my repository so I made it again with everything same. I want Microverse to forgive me for this mistake. By the way, I have added some screenshots at the end of my previous review so you can figure out the changes I've made according to the first review. 

### LINK OF MY OLD REPOSITORY AFTER FIRST SUBMISSION ###
[HAMMAS-SALEEM/Review#1](https://github.com/HAMMAS-SALEEM/Review-The-Survey-Form/pull/2)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='./style.css' rel='stylesheet'>
+    <title>Survey Form</title>
+    </head>
+    <body> 
+        <h1 id='title'>freeCodeCamp Survey Form</h1>
+        <p id='h1'>Thank you for taking time to help us improve the platform</p>
```
Comment (HAMMAS-SALEEM) [inline index.html:12]: - [ ] You should use semantic tags for the body such as `header` tag should contain your title `h1` and introductory paragraph `p` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <form id="survey-form">
+        <label for='name' id='name-label'>Name:</label>
+        <input id="name" type="text" placeholder='Enter your name'><br><br>
+        <label for='email' id='email-label'>Email:</label>
```
Comment (HAMMAS-SALEEM) [inline index.html:16]: - [ ] You should use semantic tags for the body such as the `section` tag should contain your `form` tag.

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺 #

Hi @HAMMAS-SALEEM ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️ ##

139840240-ea0d564a-492a-44e3-87af-8d5830ab0d6d

## Optional suggestions ##
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

linters error
- [ ]  Your linters are showing some errors. Fix them

## Your design ##

your design

## Freecodecamp design ##

freecodecamp design


- [ ]  I know you worked hard on your design but it is not matching with the freecodecamp template. Please follow the template

**Review by rotimiazeez — APPROVED** [review]
# Hi @HAMMAS-SALEEM , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HAMMAS-SALEEM** [comment]
Thanks for reaching out and approving my project.
Just one last thing should I fix linter errors before merging or I should
merge my branch with errors

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Pull request merged**